src/main/java/com/ita/if103java/ims/service/impl/ItemServiceImpl.java

```diff
@@ -177,11 +183,11 @@ private float getVolumeOfPassSavedItems(ItemTransactionRequestDto itemTransactio
 
 
     private boolean isLowSpaceInWarehouse(ItemTransactionRequestDto itemTransaction, Long accountId) {
-        float volume = getVolumeOfPassSavedItems(itemTransaction, accountId);
+        float volume = toVolumeOfPassSavedItems(itemTransaction, accountId);
         if (volume == 0) {
             return true;
         } else {
-            return volume * 100 / warehouseDao.findById(itemTransaction.getDestinationWarehouseId()).getCapacity() < 90.0;
+            return volume * 100 / warehouseDao.findById(itemTransaction.getDestinationWarehouseId()).getCapacity() < Float.parseFloat(maxWarehouseLoad);
```

Comment:
I will approve, but looks like you didn't read my previous comment about this line